spec/models/invoice_spec.rb

```diff
@@ -12,4 +12,11 @@
     it { should have_many(:items).through(:invoice_items) }
     it { should have_many(:merchant).through(:items) }
   end
+
+  it '#total_revenue' do
+    @merchants
+    @invoice_4
+
+    expect(@invoice_4.total_revenue).to eq(124800)
```

Comment:
View has the number_to_currency, I'd probably try and keep this test in the same format.